### PR TITLE
Enhance noop emulation to insert completion delay

### PR DESCRIPTION
**Pull request by stsoe**
Minor tweaks based on profiling cost of start() and wait().
Remove xclProbe from xrt iops test case.
Fix xcl iops test case calling unmap.
Ensure both test create same number of commands.

### CODE REVIEW

**Comment by stsoe** [comment]
@mamin506 Please check iops test case changes.  Do not worry about the other changes, which are mostly to ease profiling.

**Review by mamin506 — APPROVED** [review]

File: tests/xrt/perf_IOPS/xcl_api_iops.cpp [hunk 1]
```diff
@@ -40,18 +40,26 @@ load_file_to_memory(const std::string& fn)
     return bin;
 }
 
-double runTest(xclDeviceHandle handle, std::vector<std::shared_ptr<task_info>>& cmds,
-               unsigned int total)
+int
+startCmd(xclDeviceHandle handle, task_info* cmd)
+{
+  cmd->ecmd->state = ERT_CMD_STATE_NEW;
```
Comment (mamin506) [inline tests/xrt/perf_IOPS/xcl_api_iops.cpp:46]: KDS would set command state to ERT_CMD_STATE_NEW just before the command added to pending list.

Comment (stsoe) [reply tests/xrt/perf_IOPS/xcl_api_iops.cpp:46]: Yes, but that is specific to KDS and doesn't work in some flows that don't use KDS.

**Comment by stsoe** [comment]
Replaced by #4676

**Pull request closed without merging**